### PR TITLE
[front] - fix: radix console logs

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at fixing the radix console log errors. They should all be gone now.

**References:**
- https://github.com/dust-tt/dust/issues/8058

### CODE REVIEW

File: front/components/assistant_builder/ActionsScreen.tsx [hunk 1]
```diff
@@ -953,16 +955,16 @@ function AdvancedSettings({
   setMaxStepsPerRun: (maxStepsPerRun: number | null) => void;
 }) {
   return (
-    <Popover
-      trigger={
+    <PopoverRoot>
```
Comment (aubin-tchoi) [inline front/components/assistant_builder/ActionsScreen.tsx:958]: Why not use the Popover component directly with popoverTriggerAsChild?

Comment (JulesBelveze) [reply front/components/assistant_builder/ActionsScreen.tsx:958]: Totally forgot with have that props.. 🙈

Comment (flvndvd) [reply front/components/assistant_builder/ActionsScreen.tsx:958]: +1 with @aubin-tchoi

**Review by flvndvd — APPROVED** [review]
LGTM once comment is fixed.

**Pull request merged**